syntax = "proto3";

package login;

service LoginService {
    rpc LoginUser (LoginRequest) returns (LoginReply){};
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginReply {
    bool ok = 1;
}